.claude/skills/oba-api-review/README.md

```
@@ -34,6 +34,15 @@ That said, nothing here is specific to that project. Point `oba-api-review` at a
 /oba-api-review "remove situationIds from stops-for-route"   # a proposed change, described in English
 ```
 
+Add `--output` to also save the report as markdown, in addition to printing it inline:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)